includes/wizards/class-setup-wizard.php

```diff
@@ -653,6 +653,15 @@ public function enqueue_scripts_and_styles() {
 			$this->get_style_dependencies(),
 			NEWSPACK_PLUGIN_VERSION
 		);
+		\wp_localize_script(
+			'newspack-setup-wizard',
+			'newspack_ads_wizard',
+			array(
+				'iab_sizes'          => function_exists( '\Newspack_Ads\get_iab_sizes' ) ? \Newspack_Ads\get_iab_sizes() : [],
+				'mediakit_edit_url'  => get_option( 'pmk-page' ) ? get_edit_post_link( get_option( 'pmk-page' ) ) : '',
+				'can_connect_google' => OAuth::is_proxy_configured( 'google' ),
```

Comment:
Only `can_connect_google` is used in #2986, no need for `iab_sizes` and `mediakit_edit_url` here.

Reply:
Addressed in https://github.com/Automattic/newspack-plugin/pull/3420